Sources/LeafKit/LeafRenderer.swift

```diff
@@ -21,6 +21,21 @@ final class Cache: LeafCache {
     }
 }
 
+protocol CustomTagProtocol {
```

Comment:
Could we call this `LeafTag`? Maybe `TemplateTag`? In an end user's code, `CustomTagProtocol` seems pretty generic.